frontend/console/e2e/global-setup.ts

```diff
@@ -0,0 +1,24 @@
+import { type FullConfig, chromium } from '@playwright/test'
+
+const globalSetup = async (config: FullConfig) => {
+  console.log('Starting global setup...')
+
+  const browser = await chromium.launch()
+
+  // Create a new browser context
```

Comment:
I feel like we probably don't need this comment or the one below

Reply:
I agree 😂